server/server.js

```diff
@@ -39,6 +41,24 @@ app.get("/movies/:id", async (req, res) => {
 
 });
 
+//Get reviews, takes movieId and pageNumber as parameters.
+//calls server function loadReviewsForPageX.
+//Then sends response back to frontend
+app.get("/reviews/:id/", async (req, res) => {
+  const page = req.query.page;
+  const reviews = await loadReviews(req.params.id,page);
+  res.send(reviews)
+});
+
+//Get reviews, takes movieId and pageNumber as parameters.
+//calls server function loadReviewsForPageX.
+//Then sends response back to frontend
+app.get("/reviews/:id/", async (req, res) => {
+  const page = req.query.page;
+  const reviews = await loadReviews(req.params.id,page);
+  res.send(reviews)
+});
```

Comment:
I think this is the exact same code repeated twice, i.e. both on lines 44-51 and 53-60.